### PR TITLE
Develop changed

### PR DESCRIPTION
**Pull request by DimaCoDeeema**

### CODE REVIEW

**Review by ZhukSeeker — APPROVED** [review]

File: cypress/fixtures/example.json
Comment (djtyrf312) [inline cypress/fixtures/example.json:1]: why do you have this file here?

File: cypress/e2e/spec.cy.js
Comment (djtyrf312) [inline cypress/e2e/spec.cy.js:1]: why do you have this file here

File: cypress/support/commands.js
Comment (djtyrf312) [inline cypress/support/commands.js:1]: why do you have this file here?

File: cypress/support/e2e.js
Comment (djtyrf312) [inline cypress/support/e2e.js:1]: why do you have this file here?

File: src/pageObject.js [hunk 1]
```diff
@@ -1,15 +1,31 @@
...
+  constructor(url) {
+    this.url = url;
+    this.header = new Header();
+  };
+  clickOnLogo() {
+    return 'Click on the Conduit logo';
+  };
 };
 
 class ArticlePage extends PageObject {
```
Comment (djtyrf312) [inline src/pageObject.js:19]: ArticlePage
constructor takes url and slug. Constructor should modify original url by adding slug at the end

File: src/pageObject.js [hunk 2]
```diff
@@ -1,15 +1,31 @@
...
-
+  get commentButton() {
+    return '[Publish comment] button';
+  };
+  clickOnCommentButton() {
+    return 'Click on the [Publish comment] button';
```
Comment (djtyrf312) [inline src/pageObject.js:24]: should contain clickOnCommentButton method. The method should return a string 'Click on the' + commentButton

you need to use commentButton property here
```suggestion
    return 'Click on the [Publish comment] button';
```

**Review by djtyrf312 — CHANGES_REQUESTED** [review]
check comments, remove extra files and make fixes in the main task

**Pull request closed without merging**